src/bluetooth_numbers/reverse_lookup.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import typing
+from typing import Literal
+from uuid import UUID
+
+from bluetooth_numbers import characteristic, company, descriptor, oui, service
+
+LOGIC = Literal["OR", "AND", "SUBSTR"]
+UUID_TYPES = Literal["characteristic", "company", "descriptor", "ouis", "service"]
+UUID_TYPE_DEFAULT = ("characteristic", "company", "descriptor", "ouis", "service")
```

Comment:
The plural "ouis" seems inconsistent with the other types, which are singular.

Reply:
Agree, fixed